### DISCUSSION TITLE
Device ID changed from CTLV2 to CTLV3

### DISCUSSION THREAD

**Discussion by vonSchlau** [john30/ebusd#1329] [General]
Vaillant changed the Device ID for sensoCOMFORT VRC 720 on the ebus from CTLV2 to CTLV3. 
So ebusctl info returns: 
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV3;SW=0709;HW=3704"
therefore the automatic configuration does not work because there is no configuration file with ctlv3 in its name.
renaming a local copy to 15.ctlv3.csv seems to work ...

**Comment by dylan09** [comment]
Same for me. Also renamed the file. Most definitions are working.

**Reply by chrizzzp** [reply]
@dylan09 @vonSchlau 

If you get to decoding ebus messages from the 720/3 I would be very interested in the register for the ‘HWC alternative point’. This setting is only available in the 720/3 controller UI (not in 720/2) and I would like to check if this setting is hidden but also available in the 720/2 via ebus. This setting can be used to exclusively activate the backup heater in DHW (hot water) mode. Thanks!
